osf/management/commands/manage_switch_flags.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from django.core.management.base import BaseCommand
+
+from osf.features import switches, flags
+from waffle.models import Flag, Switch
+
+logger = logging.getLogger(__name__)
+
+def manage_waffle():
+    file_switches = [getattr(switches, switch) for switch in dir(switches) if '__' not in switch]
```

Comment:
Instead of just using the raw module `switches` I think it would be easier at this point to just make `features.py`/`switches.py`/`flags.py` etc into their own YAML or JSON dict or a configclass then import that as a list.

Reply:
I would agree, but theres many places in the codebase where these variables are referenced.